odoo/models.py

```diff
@@ -1813,7 +1814,10 @@ def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=Fals
         Get the list of records in list view grouped by the given ``groupby`` fields
 
         :param domain: list specifying search criteria [['field_name', 'operator', 'value'], ...]
-        :param list fields: list of fields present in the list view specified on the object
+        :param list fields: list of fields present in the list view specified on the object.
+                A field is either 'name' (field name, using the default aggregation),
+                or 'name:agg' (use 'agg' to aggregate values of given field name),
```

Comment:
Would be clearer to use `field` here and above, to demonstrate that the third form is a mapping/renaming.